corehq/warehouse/transforms/sql/location_dim.sql

```diff
@@ -0,0 +1,97 @@
+-- First insert all records into location dim from staging tables without foreign keys
+INSERT INTO {{ location_dim }} (
```

Comment:
upsert does require an upgrade. yea i think we can definitely work around if we don't want to upgrade. thought it was a good excuse to bump our postgres version, but may be more hassle than it's worth, especially on icds